benchmarks/converse/randomttl/randomttl.C

```diff
@@ -169,6 +163,14 @@ CmiStartFn mymain(int argc, char *argv[])
       CmiAbort("Usage: ./randomttl <k> <ttl>\n");
   }
 
+  std::srand (unsigned(std::time(0)) + CmiMyPe());
+
+  // set values to Pes
+  for (int i=0; i < CmiNumPes(); i++) CpvAccess(peVector).push_back(i);
```

Comment:
It'd be more efficient to replace this loop with the following:

CpvAccess(peVector).resize(CmiNumPes());
std::iota(CpvAccess(peVector).begin(), CpvAccess(peVector).end(), 0);

Reply:
Good point, done!